Microsoft.Toolkit.Uwp.SampleApp/SamplePages/GridSplitter/GridSplitterPage.xaml

```diff
@@ -0,0 +1,47 @@
+﻿<Page
+    x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.GridSplitterPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    mc:Ignorable="d">
+
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+        <Grid.RowDefinitions>
+            <RowDefinition></RowDefinition>
+            <RowDefinition></RowDefinition>
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition></ColumnDefinition>
+            <ColumnDefinition></ColumnDefinition>
+        </Grid.ColumnDefinitions>
+
+        <TextBlock Grid.Column="0" TextWrapping="WrapWholeWords">This text to simulate the resizing feature of the Column Resizer Control, try to change the resizer to see the effect</TextBlock>
+        <TextBlock Grid.Column="1" Grid.Row="0" TextWrapping="WrapWholeWords">This text to simulate the resizing feature of the Column Resizer Control, try to change the resizer to see the effect</TextBlock>
+
+        <TextBlock Grid.Column="0" Grid.Row="1" TextWrapping="WrapWholeWords">This text to simulate the resizing feature of the Column Resizer Control, try to change the resizer to see the effect</TextBlock>
+        <TextBlock Grid.Column="1" Grid.Row="1" TextWrapping="WrapWholeWords">This text to simulate the resizing feature of the Column Resizer Control, try to change the resizer to see the effect</TextBlock>
+
+        <!--Column Grid Splitter-->
+        <controls:GridSplitter 
+                Width="11"
+                Background="Gray"
+                Foreground="White" 
+                FontSize="11"
+                HorizontalAlignment="Right" >
+        </controls:GridSplitter>
+
+        <!--Row Grid Splitter-->
+        <controls:GridSplitter 
+            ResizeDirection="Rows"
+            Foreground="White"
+            Background="Gray" Height="11"
+            HorizontalAlignment="Stretch" 
+              VerticalAlignment="Bottom">
+        </controls:GridSplitter>
+
```

Comment:
Sorry but I didn't get what you mean, do you mean in the demonstration sample I should add the splitters in sort of Rectangle shape with 3 rows and 3 columns and the rectangle in the middle with 2 Row splitters and 2 Columns Splitters ?